Content.Server.Database/Entity/Models/Admin/AdminRank.cs

```diff
@@ -0,0 +1,23 @@
+namespace Content.Server.Database.Entity.Models
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations.Schema;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+    [Table("admin_rank")]
+    public class AdminRank : IEntityTypeConfiguration<AdminRank>
+    {
+        [Column("admin_rank_id")]
+        public int Id { get; set; }
+        [Column("name")]
+        public string Name { get; set; } = null!;
+
+        public ICollection<Admin> Admins { get; set; } = null!;
+        public ICollection<AdminRankFlag> Flags { get; set; } = null!;
+
+        public void Configure(EntityTypeBuilder<AdminRank> builder) {
```

Comment:
No need to have this configure method if you're not configuring anything, no?

Reply:
Could move the column annotations in there, though honestly I want to get rid entirely of them and lets EF Core do the naming instead. Table names are inherited from DbSet btw. It shouldn't affect anything as far as I know, but it would keep the definitions cleaner to look at.

As for empty configure method, well, not sure there's anything to do here? Not sure if skipping the TypeConfiguration subclassing will break anything or not, can try.

Reply:
Letting EFCore do naming makes typing queries for postgres a complete PITA since postgres *really* does not like uppercased names.

> Not sure if skipping the TypeConfiguration subclassing will break anything or not, can try.

It won't